src/Nox.Generator/Application/Commands/DeleteByIdCommand.template.cs

```diff
@@ -58,7 +64,7 @@ public virtual async Task<bool> Handle(Delete{{entity.Name}}ByIdCommand request,
 			var entity = await DbContext.{{entity.PluralName}}.FindAsync({{entity.Keys | array.map "Name" | keysQuery}});
 			if (entity == null{{if (entity.Persistence?.IsAudited ?? true)}} || entity.IsDeleted == true{{end}})
```

Comment:
```suggestion
			if (entity == null)
```
This has nothing to do with your PR, but just to bring it here I don't think we need this, because there is a global filter in the AppDbContext that would never return soft deleted entries.